packages/dd-trace/test/setup/core.js

```diff
@@ -9,12 +9,20 @@ const semver = require('semver')
 const platform = require('../../src/platform')
 const node = require('../../src/platform/node')
 const AsyncHooksScope = require('../../src/scope/async_hooks')
+const AsyncLocalStorageScope = require('../../src/scope/async_local_storage')
 const agent = require('../plugins/agent')
 const externals = require('../plugins/externals.json')
 
+const defaultScope = semver.satisfies(process.versions.node, '>=14.5 || ^12.9.0')
+  ? 'async_local_storage'
+  : 'async_hooks'
+
 const asyncHooksScope = new AsyncHooksScope({
   trackAsyncScope: true
 })
+const asyncLocalStorageScope = defaultScope === 'async_hooks' ? null : new AsyncLocalStorageScope({
```

Comment:
Since the condition is the same as `platform.getScope`, could that be called instead?

Reply:
I was trying to avoid using the decision-making logic from `getScope` in test files, partly because we need to test `getScope` itself.